UPGRADE-4.0.md

```diff
@@ -20,3 +20,7 @@ UPGRADE FROM 3.x to 4.0
 ### Configuration
 
 * `sonata_notification.admin.enabled` is not enabled by default
+
+### Commands
+
+They no longer extends the deprecated `ContainerAwareCommand` class. They extend `Command` class now.
```

Comment:
```suggestion
All commands now extend `Symfony\Component\Console\Command\Command` class instead of the deprecated `Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand` class.
```